ksql-serde/src/test/java/io/confluent/ksql/serde/avro/KsqlAvroDeserializerTest.java

```diff
@@ -53,236 +62,733 @@
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 import org.junit.runner.RunWith;
+import org.mockito.Mock;
 import org.mockito.junit.MockitoJUnitRunner;
 
 @RunWith(MockitoJUnitRunner.class)
 public class KsqlAvroDeserializerTest {
 
-  private static final String AVRO_SCHEMA_STRING = "{"
+  private static final org.apache.avro.Schema BOOLEAN_AVRO_SCHEMA =
+      parseAvroSchema("{\"type\": \"boolean\"}");
+
+  private static final org.apache.avro.Schema INT_AVRO_SCHEMA =
+      parseAvroSchema("{\"type\": \"int\"}");
+
+  private static final org.apache.avro.Schema LONG_AVRO_SCHEMA =
+      parseAvroSchema("{\"type\": \"long\"}");
+
+  private static final org.apache.avro.Schema DOUBLE_AVRO_SCHEMA =
+      parseAvroSchema("{\"type\": \"double\"}");
+
+  private static final org.apache.avro.Schema STRING_AVRO_SCHEMA =
+      parseAvroSchema("{\"type\": \"string\"}");
+
+  private static final org.apache.avro.Schema STRING_ARRAY_AVRO_SCHEMA =
+      parseAvroSchema("{\"type\": \"array\", \"items\": \"string\"}");
+
+  private static final org.apache.avro.Schema INT_MAP_AVRO_SCHEMA =
```

Comment:
We should also include connect's map with optional string keys representation for avro (list of key-value pairs). We have tests that cover this in the old structure (using `shouldDeserializeConnectTypeCorrectly`).

Reply:
Done in follow up PR.